x-pack/platform/plugins/shared/stack_connectors/public/connector_types/bedrock/constants.tsx

```diff
@@ -71,6 +71,24 @@ export const bedrockConfig: ConfigFieldSchema[] = [
     ),
     defaultValue: DEFAULT_BEDROCK_MODEL,
   },
+  {
+    id: 'contextWindowLength',
+    label: i18n.CONTEXT_WINDOW_LABEL,
+    isRequired: false,
+    helpText: (
+      <FormattedMessage
+        defaultMessage="Can be set to manually define the context length of the default model used by the connector. Useful for open source models or models more recent than this version of Kibana"
```

Comment:
I thought 'than this version of Kibana' sounds a bit off. Maybe user doesn't need to know that?  

```suggestion
        defaultMessage="Can be set to manually define the context length of the default model used by the connector. Useful for open source models of more recent models."
```

Also suggestion to define i18n.CONTEXT_WINDOW_HELP_TEXT since this block is repeated

Reply:
Or, to have a common constant `contextWindowField`

Reply:
> I thought 'than this version of Kibana' sounds a bit off

Agreed. Sticking to "more recent models" is probably less confusing, will adapt.